transport/http/handler_test.go

```diff
@@ -291,6 +293,57 @@ func TestHandlerInternalFailure(t *testing.T) {
 		httpResponse.Body.String())
 }
 
+func TestHandlerPanic(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	headers := make(http.Header)
+	headers.Set(CallerHeader, "somecaller")
+	headers.Set(EncodingHeader, "raw")
+	headers.Set(TTLMSHeader, "1000")
+	headers.Set(ProcedureHeader, "panic")
+	headers.Set(ServiceHeader, "fake")
+
+	request := http.Request{
+		Method: "POST",
+		Header: headers,
+		Body:   ioutil.NopCloser(bytes.NewReader([]byte{})),
+	}
+
+	rpcHandler := transporttest.NewMockHandler(mockCtrl)
+	rpcHandler.EXPECT().Handle(
+		transporttest.NewContextMatcher(t, transporttest.ContextTTL(time.Second)),
+		transport.Options{},
+		transporttest.NewRequestMatcher(
+			t, &transport.Request{
+				Caller:    "somecaller",
+				Service:   "fake",
+				Encoding:  raw.Encoding,
+				Procedure: "panic",
+				Body:      bytes.NewReader([]byte{}),
+			},
```

Comment:
Instead of all this, can't you simply create an `Inbound` which you `Start()`
with a `Handler` that always panics? And use an `Outbound` to call it rather
than an HTTP client?